test/context-config.test.js

```diff
@@ -41,35 +41,35 @@ test('config', t => {
     handler
   })
 
-  fastify.inject({
+  let response = await fastify.inject({
     method: 'GET',
-    url: '/get'
-  }, (err, response) => {
-    t.error(err)
-    t.equal(response.statusCode, 200)
-    t.same(JSON.parse(response.payload), Object.assign({ url: '/get', method: 'GET' }, schema.config))
+    url: '/route'
   })
 
-  fastify.inject({
+  t.assert.ifError(response.error)
```

Comment:
All the `ifError` condition are confusing and we can remove it (and the `t.plan()` needs to be decreased accordingly

```suggestion
```

Reply:
Thanks for pointing that out. I’ve removed the `ifError` conditions and adjusted `t.plan()` accordingly